Fabric.Databus.API/Dockerfile

```diff
@@ -9,4 +9,9 @@ RUN /usr/sbin/update-ca-certificates
 
 RUN mkdir -p /home/hcadmin/json
 
+ENV Authority=https://fabricservices.hqcatalyst.local/identity
```

Comment:
It might be better to not specify these and simply let whomever is starting the container specify them. If you wanted to provide sensible defaults, it might be better to put those in appsettings.json, as a developer will probably look there first to figure out what the config is. Docker environment variables are really there to override defaults.

Reply:
Good call - changing now...